include/oneapi/dpl/pstl/glue_algorithm_impl.h

```diff
@@ -655,6 +667,19 @@ partition_copy(_ExecutionPolicy&& __exec, _ForwardIterator __first, _ForwardIter
                                                              __first, __last, __out_true, __out_false, __pred);
 }
 
+namespace __internal
+{
+struct __sort_fn
```

Comment:
As an option for such identical call wrapper we can use one following solution (to have less code lines)
```
#define ONEDPL_CALL_ALGO(std_algo_name) \
struct std_algo_name##_fn \
{ \
    void operator()(auto&&... __args) const \
    { \
        return std_algo_name(std::forward<decltype(__args)>(__args)...); \
    } \
};
```

Reply:
From my point of view, we should minimize macros  usages for these cases.
If we haven't good common place for this `struct __sort_fn` I would prefer duplicate it instead macro usage.

Reply:
I would prefer to have less code lines when we can do it w/o making code more complicated.

Reply:
1. The macro which you propose isn't the same like this predicate: we doesn't `forward` arguments in it. And in thi source lambda here - too.
2. The macro which you propose written with errors:
```C++
ONEDPL_CALL_ALGO(std::sort)
```

will expands to
```C++
void
operator()(auto&&... __args) const
{
    return std::sort(__args...);
}
```

Reply:
I dont insist on. Especially since 1) for iterator based sort/stable sort we forward the args by copy, 2) for range based sort/stable sort we forward the args by forward ref.